server/hydrogen-render/1-render-hydrogen-to-string.js

```diff
@@ -60,12 +61,16 @@ async function renderHydrogenToString(options) {
         } else {
           let extraErrorsMessage = '';
           if (childErrors.length > 1) {
-            extraErrorsMessage = ` (somehow we saw ${childErrors.length} errors but we really always expect 1 error)`;
+            extraErrorsMessage = ` (somehow we saw ${
+              childErrors.length
+            } errors but we really always expect 1 error)\n${childErrors
+              .map((childError, index) => ` ${index}. ${childError.message} ${childError.stack}`)
+              .join('\n')}`;
           }
 
           const error = new RethrownError(
             `Child process failed with exit code ${exitCode}${extraErrorsMessage}`,
-            childErrors[0]
+            childErrors[0] || new Error('No child errors')
```

Comment:
This is possible if the process exits for some other random reason. Exit code can even by `null`.